scripts/README.v1.1.sh

```diff
@@ -0,0 +1,90 @@
+###############################################################################
```

Comment:
A README shell script?  Maybe it should then default to NOT building the calling regions unless you pass it some argument.

Reply:
This is a modified version of Yossi's script -- for the record, I'm not the one who named it readme.  I will change it.

Reply:
Agreed the name of this could be clearer :)